client/widgets/Buttons.cpp

```diff
@@ -23,6 +23,8 @@
 #include "../windows/InfoWindows.h"
 #include "../../lib/CConfigHandler.h"
 
+using namespace std;
```

Comment:
Global `using` is not recommended.

Reply:
Tbh I think we had long talks about it on Slack so I strongly against ```using``` for something as common as ```std```.

Reply:
Ok, that's fine.